### ISSUE TITLE
Request: add text to clarify that visibility of app can be toggled

### ISSUE THREAD

**Issue by mjanssens** [juliushaertl/apporder#58]
Hi,
Your app is not just about the app order, but gives the ability to make an app visible or not.
I didn't notice that as the title of your app doesn't imply that, so does your app have the right title?

But i think its a great extra feature, it's worth to mention it in the app info and in the store I guess.
Some users would want to disable apps from their view if they don't use them, although they are installed.

**Comment by juliushaertl** [comment]
Good point. Are you up to open a pull request to add some notice about the app hiding feature in the README / in the settings template (https://github.com/juliushaertl/apporder/blob/master/templates/admin.php) ?

**Comment by mjanssens** [comment]
Never done a pull request, but willing to learn how to.
Maybe I will investigate in the coming holidays ;-)
